docs/usage/quickstart.rst

```diff
@@ -3,112 +3,112 @@
 
-First, create a directory, e.g. :file:`inventory/quickstart-cluster`, in which
-the inventory will be stored. For our setup, we need to create two files. One
-listing all the hosts, aptly called :file:`hosts`:
+To create an inventory:
+
+1. Create a directory (for example,
+:file:`inventory/quickstart-cluster`) in which the inventory will be stored.
 
-.. code-block:: ini
+2. Create two files:
 
-    node-01 ansible_host=10.0.0.1 ansible_user=centos
-    node-02 ansible_host=10.0.0.2 ansible_user=centos
-    node-03 ansible_host=10.0.0.3 ansible_user=centos
+  - The first, called :file:`hosts`:, lists all hosts::
 
-    [kube-master]
-    node-01
-    node-02
-    node-03
+         node-01 ansible_host=10.0.0.1 ansible_user=centos
+         node-02 ansible_host=10.0.0.2 ansible_user=centos
+         node-03 ansible_host=10.0.0.3 ansible_user=centos
 
-    [etcd]
-    node-01
-    node-02
-    node-03
+         [kube-master]
+         node-01
+         node-02
+         node-03
 
-    [kube-node]
-    node-01
-    node-02
-    node-03
+         [etcd]
+         node-01
+         node-02
+         node-03
 
-    [k8s-cluster:children]
-    kube-node
-    kube-master
+         [kube-node]
+         node-01
+         node-02
+         node-03
 
-Make sure to change IP-addresses, usernames etc. according to your
-infrastructure.
+         [k8s-cluster:children]
+         kube-node
+         kube-master
 
-In a second file, called :file:`kube-node.yml` in a :file:`group_vars`
-subdirectory of our inventory, we declare how to setup storage (in the
-default configuration) on hosts in the *kube-node* group, i.e. hosts on which
-Pods will be scheduled:
+    Make sure to change IP addresses, user names, etc. to conform to your
+    infrastructure.
 
-.. code-block:: yaml
+  - A second file, called :file:`kube-node.yml`, in a :file:`group_vars`
+   subdirectory of the inventory, declares how to set up storage (in the
+   default configuration) on hosts in the *kube-node* group, that is,
+   hosts on which Pods will be scheduled::
 
-    metal_k8s_lvm:
-      vgs:
-        kubevg:
-          drives: ['/dev/vdb']
+         metal_k8s_lvm:
+           vgs:
+             kubevg:
+               drives: ['/dev/vdb']
 
-In the above, we assume every *kube-node* host has a disk available as
+In the above, every *kube-node* host is assumed to have a disk available as
 :file:`/dev/vdb` which can be used to set up Kubernetes *PersistentVolumes*. For
 more information about storage, see :doc:`../architecture/storage`.
 
 Entering the MetalK8s Shell
 ---------------------------
-To easily install a supported version of Ansible and its dependencies, as well
-as some Kubernetes tools (:program:`kubectl` and :program:`helm`), we provide a
-:program:`make` target which installs these in a local environment. To enter this
+To install a supported version of Ansible and its dependencies, along with some
+Kubernetes tools (:program:`kubectl` and :program:`helm`), create a
```

Comment:
No. We provide a `make` target, the user has not to 'create one'.

Reply:
rolled back.